remove bogus autogenerated '' set"other '' methods that are actually the < and >
operators, not methods - the bogus syntax (quote in identifier) makes the
Byte-compiling python fails on this file at package installation.

source: https://src.fedoraproject.org/rpms/kdevelop-python/raw/master/f/kdev-python-1.7.0-py3-doc-syntax.patch

--- a/documentation_files/__builtin_types__.py
+++ b/documentation_files/__builtin_types__.py
@@ -44,14 +44,6 @@
 		"""
 		pass
 		
-	def set"other(self, ():
-		"""
-		Test whether the set is a true subset of *other*, that is,
-		``set <= other and set != other``.
-		
-		"""
-		pass
-		
 	def issuperset(self, other):
 		"""set >= other
 		
@@ -59,14 +51,6 @@
 		
 		"""
 		pass
-		
-	def set"other(self, ():
-		"""
-		Test whether the set is a true superset of *other*, that is, ``set >=
-		other and set != other``.
-		
-		"""
-		pass
 		
 	def union(self, other,more):
 		"""set | other | more
